server/base/src/main/java/org/apache/accumulo/server/metadata/ServerAmpleImpl.java

```diff
@@ -377,14 +377,11 @@ public void deleteScanServerFileReferences(String serverAddress, UUID scanServer
     Objects.requireNonNull(scanServerLockUUID, "Server uuid must be supplied");
     try (
         Scanner scanner = context.createScanner(DataLevel.USER.metaTable(), Authorizations.EMPTY)) {
-      scanner.setRange(ScanServerFileReferenceSection.getRange());
-      scanner.fetchColumn(new Text(serverAddress), new Text(scanServerLockUUID.toString()));
+      scanner
+          .setRange(new Range(ScanServerFileReferenceSection.getRowPrefix() + scanServerLockUUID));
```

Comment:
If we still need to insert old style mutations, do we still need to look for them here?

Reply:
We don't need to write the old mutation style for puts, just for putDeletes. 

This code path is only called from the ScanServer when removing its own UUID reference on shutdown. 

Any Scan Servers that are writing the old mutation style are running an older version of accumulo code and wouldn't be using this code path. 

We only need to look in both places for the manager code's cleanup  operation (ScanServerMetadataEntries).